spec/person_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../person'
+
+describe Person do
+  context 'When testing person class' do
+    person = Person.new('24', 'Tash', true)
+
+    it 'should display name and age ' do
+      expect(person).to be_an_instance_of Person
+    end
+    it 'Person name ' do
+      expect(person.name).to eq 'Tash'
+    end
+    it 'person age' do
+      expect(person.age).to eq '24'
+    end
+    it 'Person parent permission' do
+      expect(person.parent_permission).to be true
+    end
+  end
```

Comment:
- [ ] A person class has some methods which aren't tested, for example:

```Ruby
 def add_rental(book, date)
    Rental.new(date, self, book)
  end

  def correct_name
    @name
  end

  def of_age?
    @age >= 18
  end
```

- [ ] The `can_use_services?` method should be tested twice when parental permission is true and when it is false.

```Ruby
# Something like this below

 ```ruby
  it "The can_use_services? method returns false if under age and no parent permission" do
      person = Person.new(14, "Rose", false)
      expect(person.can_use_services?).to be false
    end
    
    it "The can_use_services? method returns true if has parent permission" do
      person = Person.new(14, "Rose", true)
      expect(person.can_use_services?).to be true
    end
  end
```